src/instrument/devices/tetramm_picoammeter.py

```diff
@@ -55,9 +55,9 @@ def __init__(self, *args, port_name="TetrAMM", **kwargs):
         self.current4.mean_value.kind = "hinted"
 
     def trigger(self):
-        '''
+        """
```

Comment:
Expect this change will also appear in other PRs.  The CI workflow fails without it.